prepare_release.sh

```diff
@@ -0,0 +1,139 @@
+#!/bin/bash
+
+set -eo pipefail
+
+mute=">/dev/null 2>&1"
+version="$1"
+is_hotfix=false
+
+if ! [[ $common_sh ]]; then
+	cwd="$(dirname "${BASH_SOURCE[0]}")"
+    source "${cwd}/scripts/helpers/common.sh"
```

Comment:
`./prepare_release.sh: line 11: ./scripts/helpers/common.sh: No such file or directory`

Did you forget to add common.sh?